examples/get_event_messages.c

```diff
@@ -38,12 +41,19 @@ void callback(event_client_event_t *event) {
       event_subscribe(event->client, NULL, TOPIC_MS_CONFIRMED, 1);
       event_subscribe(event->client, NULL, TOPIC_MS_REFERENCED, 1);
       event_subscribe(event->client, NULL, TOPIC_MESSAGES, 1);
+      event_subscribe(event->client, NULL, TOPIC_MS_MILESTONE, 1);
+      event_subscribe(event->client, NULL, TOPIC_MS_TRANSACTION, 1);
+      event_subscribe(event->client, NULL, TOPIC_MS_TAGGED_DATA, 1);
+      event_sub_tx_msg_tagged_data(event->client, NULL, test_tag, 1);
+      event_sub_msg_tagged_data(event->client, NULL, test_tag, 1);
+      event_sub_txn_included_msg(event->client, NULL, test_transaction_id, 1);
       event_subscribe_msg_metadata(event->client, NULL, test_message_id, 1);
-      event_sub_address_outputs(event->client, NULL, test_bech32, true, 1);
-      event_sub_address_outputs(event->client, NULL, test_ed25519, false, 1);
       event_sub_outputs_id(event->client, NULL, test_output_id, 1);
-      event_sub_txn_included_msg(event->client, NULL, test_transaction_id, 1);
-      event_sub_msg_indexation(event->client, NULL, test_index, 1);
+      event_sub_outputs_unlock_address(event->client, NULL, "address", test_bech32, 1);
+      event_sub_outputs_unlock_address_spent(event->client, NULL, "address", test_bech32, 1);
+      event_sub_outputs_alias_id(event->client, NULL, alias_id, 1);
+      event_sub_outputs_nft_id(event->client, NULL, nft_id, 1);
+      event_sub_outputs_foundry_id(event->client, NULL, foundry_id, 1);
```

Comment:
the return value should be checked for making sure the subscription has no problems

Reply:
Checking the return value and printing respective error message.